lib/internal/modules/cjs/loader.js

```diff
@@ -1978,6 +1963,19 @@ function createRequire(filename) {
   return createRequireFromPath(filepath);
 }
 
+/**
+ * Checks if a path is relative
+ * @param {string} path the target path
+ * @returns {boolean} true if the path is relative, false otherwise
+ */
+function isRelative(path) {
+  return path === '.' || path === '..' ||
+  StringPrototypeStartsWith(path, './') ||
+  StringPrototypeStartsWith(path, '../') ||
+  ((isWindows && StringPrototypeStartsWith(path, '.\\')) ||
+  StringPrototypeStartsWith(path, '..\\'));
```

Comment:
I realize that charcode comparisons might be more performant, but I believe that this is leagues better in terms of readability, I'm happy to go with charcode comparisons if perf is a concern here

Reply:
You could probably save some cycle by returning early if `path.length && path[0] === '.'` is false

Reply:
oh yeah of course! 🙂 

Added! thanks for the suggestion 🙂